region/lowernorfair/east.json

```diff
@@ -1592,7 +1592,15 @@
           "to": [
             {
               "id": 2,
-              "requires": null,
+              "requires": [
```

Comment:
I frequently see people with Gravity going through the acid, but I don't know if it's just a speed strat. Did you check that one for cost-effectiveness?

Reply:
Definitely a speed strat